Octokit.Tests/Clients/GistsClientTests.cs

```diff
@@ -1,321 +1,573 @@
 ﻿using NSubstitute;
-using Octokit;
 using Octokit.Internal;
 using System;
 using System.Collections.Generic;
 using System.Net;
 using System.Threading.Tasks;
 using Xunit;
 
-public class GistsClientTests
+namespace Octokit.Tests.Clients
```

Comment:
I see, I need to get use to this ugly underline then :/
![image](https://cloud.githubusercontent.com/assets/2827220/14525975/2d75dab0-0238-11e6-97de-48c3a298688f.png)